selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/PullRequestPanel.java

```diff
@@ -231,8 +237,11 @@ public void waitRepoUrl(String url) {
   }
 
   public void openPullRequestOnGitHub() {
-    loadPageDriverWait
-        .until(elementToBeClickable(By.xpath(PullRequestLocators.OPEN_GITHUB_BTN)))
-        .click();
+    Wait<WebDriver> wait =
```

Comment:
I suggest you to use existed method [waitElementIsRendered(WebElement webElement)](https://github.com/eclipse/che/blob/master/selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/TestWebElementRenderChecker.java#L54), like it was done [here](https://github.com/eclipse/che/blob/master/selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java#L602).